extensions/amp-carousel/0.1/base-carousel.js

```diff
@@ -63,11 +63,6 @@ export class BaseCarousel extends AMP.BaseElement {
   }
 
   /** @override */
-  prerenderAllowed() {
```

Comment:
Why is this removed? There's a trick in carousel. Basically, carousel says, don't touch much children, I will call "prerender" and "render" manually. This is way it wants to get these signals itself.